app/controllers/drafts_controller.rb

```diff
@@ -33,7 +33,7 @@ def set_record
     if(@draft.record)
       @record = @draft.record
     else
-      @record = @draft.info["record_type"].constantize.new
+      @record = @draft.record_type.constantize.new
```

Comment:
I'm confused by the two fields that mean the same thing. Is this the one that we should be using going forward, and info['record_type'] is just for legacy reasons?

Reply:
Drafts for shelter updates have a related record.  Drafts for new shelters have a record_type field that is used to create a new object of that type.

Reply:
I think the @draft.info was a mistake since the info field has the shelter data